src/features/questions/ui/button/SendButton.tsx

```
@@ -0,0 +1,46 @@
+'use client'
+
+import { useFormContext } from 'react-hook-form'
+import { useFormStatus } from 'react-dom'
+import { useRouter } from 'next/navigation'
+import { MessageFormValues } from '@/features/message/_send/model/formSchema'
+import { usePostSendMessage } from '@/features/message/_send/api/sendMutation'
+import { Button } from '@/shared/components/Button'
+
+export default function SendButton({}) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)